CGPs/cgp-0068/mainnet.json

```diff
@@ -1,7 +1,7 @@
 
   [
     {
-      "contract": "StableToken", 
+      "contract": "GoldToken", 
       "function": "approve",
       "args": [
         "0xc59533fd060e38230100F3577eF5ad9467ae2594",
```

Comment:
You are likely aware but this would give you spend approval of 428,000 CELO which is a floating rate(1 CELO = 0.81 $cUSD - 16/02/23 9:08 PST). As your milestones and KPI's are all denominated in $cUSD this current ask would give ~20% less than you need. This may be ok depending on your budget and the leeway of expenses but something to consider.

I could see a couple of options that would assist:

- Request a higher amount than what is needed and stipulate in the proposal that the CELO would be converted ASAP after approval and the remainder would be sent back to the governance contract.
- Leave as is and adjust the budget accordingly after the approval and exchange.

However, any changes made should be stipulated in the proposal.

Reply:
Noted, thank you 
I would do that right away

Reply:
Hi @0xzoz kindly check my new pr, we decided to ask for celo and refund the extra back to the governance contract after converting our initial proposal amount to cUSD

Reply:
This looks better @Khadeeejah. There is still an issue unfortunately, the new PR only requests 550 Celo and not 550, 000 due to the second argument missing 3 zeros.

Reply:
Oh wow, let me look at that right away, it's meant to be 550,000 and 18 zeros right?